schemas/deviceToCloud/gps/gps.json

```diff
@@ -12,8 +12,12 @@
              "messageType":{
                 "type":"string",
                 "const":"HELLO"
```

Comment:
Interesting -- I never noticed this "HELLO" before.  I do not think current firmware ever sends this, but I suppose a customer could add it.  What does it do on the back end / front end if the device sends "HELLO"?

Reply:
See comment [here](https://github.com/nRFCloud/application-protocols/pull/28#discussion_r747124545)